src/flow-control/log-timings.js

```diff
@@ -0,0 +1,57 @@
+const Rx = require('rxjs');
+const _ = require('lodash');
+const { bufferCount, take } = require('rxjs/operators');
+const BaseHandler = require('./base-handler');
+
+module.exports = class LogTimings extends BaseHandler {
+    printExitInfoTimingTable(exitInfos) {
+        const exitInfoTable = _(exitInfos)
+            .map(({ command, timings, index, killed, exitCode }) => {
+                const readableDurationMs = (timings.endDate - timings.startDate).toLocaleString();
+                return {
+                    'call-index': index,
+                    name: command.name,
+                    duration: `~${readableDurationMs}ms`,
+                    'exit-code': exitCode,
+                    killed,
+                    command: command.command,
+                };
+            })
+            .sortBy('duration')
+            .value();
+
+        console.log('\nTimings:');
+        console.table(exitInfoTable);
+        return exitInfos;
+    };
+
+    handle(commands) {
+        if (!this.logger) { return {commands}; }
+
+        // individual process timings
+        commands.forEach(command => {
+            command.timer.subscribe( {
+                next: ({startDate, endDate}) => {
+                    if (!endDate) {
+                        this.logger.logCommandEvent( `${ command.command } started at ${ startDate.toLocaleString() }`, command );
```

Comment:
Wondering if we should use the format specified in `--timestamp-format` instead (hint, date-fns is a dependency!).

Might make _part_ of the test simpler/more stable on diverse machines due to all those regular expressions...

Reply:
good idea, I'll look into it